src/pyuvsim/simsetup.py

```diff
@@ -2145,13 +2139,6 @@ def _complete_uvdata(uv_in, inplace=False, check_kw=None):
     uv_obj.flag_array = np.zeros(_shape, dtype=bool)
     uv_obj.nsample_array = np.ones(_shape, dtype=float)
 
-    uv_obj.extra_keywords = {}
```

Comment:
I do not think we should be re-setting these because they actually contain useful info like what the input yaml files are.

Reply:
Note that this change led to a number of test changes but I think it's the right thing to do.